_docs/technical/2017-01-15-cli-options.md

```diff
@@ -0,0 +1,727 @@
+~~~
+
+## `cardano-dht-keygen`
+
+An executable `cardano-dht-keygen` is a generator of random key for Kademlia DHT.
+
+`cardano-dht-keygen` hasn't any options, only 1 mandatory parameter `NONCE`, 14-characters string.
+
+Example of `cardano-dht-keygen` command:
+
+~~~
+cardano-dht-keygen 00000000000001
+~~~
+
+Example of result:
+
+~~~
+Y94xf23cG8as4ycA9R-cvo9zMDAwMDAwMDAwMDAwMDE=
+~~~
+
+Please make sure that nonce' lenght must be exactly 14 characters.
+
+## `cardano-wallet`
+
+An executable `cardano-wallet` is a CLI-based wallet + node. You can think of `cardano-wallet` as of node wrapped by wallet's functionality (without Daedalus).
+
+Usage:
+
+~~~
+cardano-wallet [OPTIONS] COMMAND
+~~~
+
+Available options:
+
+1.  `-h` and `--help`
+    <br/>Show this help info and exit.
+    <br/>&nbsp;
+2.  `--db-path FILEPATH`
+    <br/>Path to the directory with wallet's database.
+    <br/>Specified path must exist.
+    <br/>Default value: none.
+    <br/>&nbsp;
+3.  `--rebuild-db`
+    <br/>If the wallet's database already exists, discard its contents and create a new one from scratch.
+    <br/>&nbsp;
+4.  `--listen IP:PORT`
+    <br>Specify IP-address and port node uses to listen requests.
+    <br/>Default value: `0.0.0.0:24961`.
+    <br/>**Please make sure these IP-address and port are accessible, otherwise work of the node isn't guaranteed**.
+    <br/>&nbsp;
+5.  `--peer HOST:PORT/HOST_ID`
+    <br/>Peer to connect to for initial peer discovery.
+    <br/>**This option can be used more than once.**
+    <br/>Example: `--peer localhost:1234/dYGuDj0BrJxCsTC9ntJE7ePT7wUoVdQMH3sKLzQD8bo=`
+    <br/>Default value: none.
+    <br/>&nbsp;
+6.  `--peers-file FILEPATH`
+    <br/>Path to file with peers list (peers must be separated by newlines).
+    <br/>Default value: none.
+    <br/>&nbsp;
+7.  `--log-config FILEPATH`
+    <br/>Path to logger configuration file.
+    <br/>Default value: [hardcoded default logger configuration](https://github.com/input-output-hk/cardano-sl/blob/eb74d35c50603b0ca8a967f6173b61affd1e2982/src/Pos/CLI.hs#L107).
+    <br/>&nbsp;
+8.  `--logs-prefix FILEPATH`
+    <br/>Prefix to logger output path.
+    <br/>Specified path must be valid.
+    <br/>Default value: none.
+    <br/>&nbsp;
+9.  `--ssc-algo ALGO`
+    <br>Shared Seed Calculation algorithm which nodes will use.
+    <br>Possible values: `GodTossing` and `NistBeacon`.
+    <br/>Default value: `GodTossing`.
+    <br/>&nbsp;
+10. `--report-server URL`
+    <br/>URL of reporting server to send crash/error logs on.
+    <br/>**This option can be used more than once.**
+    <br/>Example: `--report-server https://my.report.server.com:4000`.
+    <br/>&nbsp;
+11. `--update-server URL`
+    <br/>URL of the server to download updates from.
+    <br/>**This option can be used more than once.**
+    <br/>Example: `--update-server https://my.update.server.com:3000`.
+    <br/>&nbsp;
+12. `--flat-distr (INT,INT)`
```

Comment:
Same here. It is intuitive by its name, but it would be good to briefly explain what a "flat" distribution is. Do it at the first occurence so you won't have to repeat yourself.